.github/workflows/merge.yml

```
@@ -18,6 +18,9 @@ jobs:
   publishing:
     name: Build and publish Python 🐍 distributions 📦 to PyPI
     runs-on: ubuntu-latest
+    environment: pypi
+    permissions:
+        id-token: write
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)